tasks/ods-build-go.yaml

```diff
@@ -0,0 +1,185 @@
+
+# Source: tasks/templates/task-ods-build-go.yaml
```

Comment:
Are lines 1 and 2 needed?

Reply:
No they are not needed. I actually missed the empty first line when committing the files.

Those lines are rendered by `helm template`, and they are not cleaned up when we cut the output and store each task in a separate file. I could remove the first line. The second line actually is kind of correct, the source is `deploy/ods-pipeline/charts/templates/task-ods-build-go.yaml`. It just doesn't render the directory containing the chart ...